core/templates/dev/head/domain/dashboard/ShowExplorationTitleSection.js

```diff
@@ -0,0 +1,18 @@
+oppia.directive('explorationTitleSection', function($compile) {
+  return {
+    restrict: 'E',
+    scope: {
+      getExploration: '&exploration',
+      activeExplorationId: '='
```

Comment:
@Arunabh98 Sorry, I've been busy now a days (and will continue to be until 26th at least). About the arrow not showing up when using `&` instead of `=` , it's most probably since the value of `activeExplorationId` is being dynamically set each time, while the arguments to the directive are evaluated only once when passed with `&`. I'm fine with using `=` to overcome this (the other solution might have been to add a manual watcher to the scope variable).